tools/plugin-toolkit/src/listCmds/actionConfig.ts

```diff
@@ -0,0 +1,47 @@
+// (C) 2021 GoodData Corporation
+import { createWorkspaceTargetConfig, WorkspaceTargetConfig } from "../_base/workspaceTargetConfig";
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { ActionOptions } from "../_base/types";
+import { createBackend } from "../_base/backend";
+import ora from "ora";
+import { asyncValidOrDie, createWorkspaceValidator } from "../_base/inputHandling/validators";
+
+export type ListCmdActionConfig = WorkspaceTargetConfig & {
+    backendInstance: IAnalyticalBackend;
+};
+
+async function doAsyncValidations(config: ListCmdActionConfig) {
```

Comment:
wouldn't be better to name the function something like `validateAsync` instead of `doAsyncValidations`?